sfge/src/test/java/com/salesforce/rules/unusedmethod/ConstructorsTest.java

```diff
@@ -149,28 +137,50 @@ public void constructorUncalledByGrandchildSuper_expectViolation(Integer arity)
         assertViolations(sourceCodes, assertion);
     }
 
+    /**
+     * Simple tests for cases where one class's constructor is called somewhere in another class.
+     *
+     * @param arity - The arity of the tested constructor.
+     */
+    @ValueSource(ints = {0, 1})
+    @ParameterizedTest(name = "{displayName}: Arity {0}")
+    public void constructorCalledViaNew_expectNoViolation(Integer arity) {
```

Comment:
New test for simply calling a constructor for one class in another class. This sort of thing is arguably covered by the tests for inner classes, but I felt that having an explicit test for it did no harm.